website/docs/r/nat_dnat_rule_v2.html.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: "flexibleengine"
+page_title: "FlexibleEngine: flexibleengine_nat_dnat_rule_v2"
+sidebar_current: "docs-flexibleengine-resource-nat-dnat-rule-v2"
+description: |-
+
+---
+
+# flexibleengine\_nat\_dnat
+
+
+## Example Usage
+
+### Dnat
+
+```hcl
+resource "flexibleengine_nat_dnat_rule_v2" "dnat" {
+  floating_ip_id = "bf99c679-9f41-4dac-8513-9c9228e713e1"
+  nat_gateway_id = "bf99c679-9f41-4dac-8513-9c9228e713e1"
```

Comment:
better to use a different uuid with floating_ip_id :P

Reply:
Done